src/components/questions/question.js

```diff
@@ -1,26 +1,50 @@
 import React from "react";
 import AnswerList from "./answersList"
+import Modal from "../Modal.jsx"
+import AddAnswer from "./addAnswer.js"
 
 
 class Question extends React.Component{
   constructor(props){
     super(props);
-    this.state = {};
+    this.state = {
+      showModal: false
+    };
+    this.helpfulClick = this.helpfulClick.bind(this);
+    this.addAnswerClick = this.addAnswerClick.bind(this);
+  }
+
+  helpfulClick(event){
+    console.log("***HELPFUL CLICK***")
+  }
+
+  addAnswerClick(event){
+    this.setState({showModal:!this.state.showModal})
   }
 
   render(){
     return (<div className="container">
       <div className="container">
         <div className="title is-inline-block">{`Q: ${this.props.question_body}`}</div>
-        <div className="is-pulled-right is-inline-block">Helpful?({this.props.question_helpfulness}){" | "}Add Answer</div>
+        <div className="is-pulled-right is-inline-block">
```

Comment:
I have used is-pulled-right before for certain things. Possibly look into different solution as I noticed some unexpected styling behavior when I used it.

Reply:
An alternative may be doing the right alignment in a separate css file and importing it.